EIPS/eip-reduce_block_reward.md

```diff
@@ -0,0 +1,38 @@
+## Preamble
+
+    EIP: <to be assigned>
```

Comment:
Please use 858, and move this file to `EIPS/eip-858.md`.

Reply:
ok done. thanks.